src/RoutingControllersOptions.ts

```diff
@@ -20,6 +20,11 @@ export interface RoutingControllersOptions {
     routePrefix?: string;
 
     /**
+     * Enable automatic fallthrough from action handlers to future actions or middleware
+     */
+    automaticFallthrough?: boolean;
```

Comment:
`fallthrough` is better then `automaticFallthrough `. Also TBH `fallthrough` does not indicate for me what this option does, but I can't find a good name for it. Maybe this terminology is highly used, simply Im not familiar with it, adding other thoughts @19majkel94 @NoNameProvided  . Is this word better then something like `callNext: boolean` ?